### PR TITLE
docs: last storage plugin is default - spaces

### PR DESCRIPTION
**Pull request by richardwardza**
Add caution that only the last configured storage plugin will be used

(Apologies for the 3 separate PR’s, I don’t have access to a proper terminal right now and doing this from the website “edit this page” link)

### CODE REVIEW

File: docs/content/add-plugins/spaces.md [hunk 1]
```diff
@@ -53,6 +53,12 @@ Make sure to use an environment variable for the secret key in a live environmen
 
 :::
 
+:::caution
+
+ When you have multiple storage plugins configured, the one declared last in the `medusa-config.js` file will be used.
```
Comment (shahednasser) [inline docs/content/add-plugins/spaces.md:58]:
```suggestion
If you have multiple storage plugins configured, the last plugin declared in the `medusa-config.js` file will be used.
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Just a small change to the wording 🙂

**Comment by richardwardza** [comment]
Thanks @shahednasser - I've done the update on the 3 storage plugin pages.

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks! @richardwardza

**Pull request merged**